### PR TITLE
Remove old Wazuh-api service references

### PR DESCRIPTION
**Pull request by CarlosRS9**
Hello team,

This PR is related to [#5824](https://github.com/wazuh/wazuh/pull/5868). The wazuh-api service has been merged together with wazuh-manager service here. This PR removes any reference to the old wazuh-api service.

### CODE REVIEW

File: source/installation-guide/installing-wazuh-manager/linux/opensuse/wazuh_server_sources_opensuse.rst [hunk 1]
```diff
@@ -104,15 +104,15 @@ To check the API service status use the following command:
 
     .. code-block:: console
 
-      # systemctl status wazuh-api
+      # systemctl status wazuh-manager
```
Comment (davidjiglesias) [inline source/installation-guide/installing-wazuh-manager/linux/opensuse/wazuh_server_sources_opensuse.rst:107]: Do we need this code? It says to check the API service status, there is not API service. Please review rest of the documentation for API service references.

File: source/learning-wazuh/vuln-detection.rst [hunk 2]
```diff
@@ -326,19 +326,19 @@ The results should look like this:
       [root@wazuh-manager centos]# sed -i 's/config.experimental_features  = false/config.experimental_features  = true/g' /var/ossec/api/configuration/config.js
 
 
-5. Restart the Wazuh API service:
+5. Restart the Wazuh API using the Wazuh-manager service:
```
Comment (davidjiglesias) [inline source/learning-wazuh/vuln-detection.rst:329]: I see a reference to config.js 3 lines above this one, please review this part deeper. Look for config.js and nodejs/npm references in the entire documentation.

File: source/upgrade-guide/upgrading/latest_wazuh3_minor.rst [hunk 3]
```diff
@@ -39,20 +39,20 @@ a) CentOS/RHEL/Fedora:
 
 .. code-block:: console
 
-    # yum upgrade wazuh-manager wazuh-api
+    # yum upgrade wazuh-manager
```
Comment (davidjiglesias) [inline source/upgrade-guide/upgrading/latest_wazuh3_minor.rst:42]: Reference to the API in line 35

File: source/user-manual/api/configuration.rst [hunk 4]
```diff
@@ -58,19 +58,19 @@ Here are all the settings available for the `api.yaml` configuration file. For m
 
     If running a cluster, the master will NOT send its local API configuration file to the workers. Each node provides its own API, if the configuration file is changed in the master node, the user should manually update the workers API configuration in order to use the same configuration. Take care of not overwriting the IP and port in the local configuration of each worker.
 
-Make sure to restart wazuh-api service after editing the configuration file:
+Make sure to restart wazuh-manager service after editing the configuration file:
```
Comment (davidjiglesias) [inline source/user-manual/api/configuration.rst:61]:
```suggestion
Make sure to restart the Wazuh API using wazuh-manager service after editing the configuration file:
```

**Review by davidjiglesias — CHANGES_REQUESTED** [review]

File: source/installation-guide/installing-wazuh-manager/linux/opensuse/wazuh_server_sources_opensuse.rst (hunk 1 above)
Comment (CarlosRS9) [reply source/installation-guide/installing-wazuh-manager/linux/opensuse/wazuh_server_sources_opensuse.rst:107]: No, this is no longer required. As you can see in the installation guides for other platforms, this fragment must be removed. I will remove it from this page as well and review if it is present in any other page.

**Review by davidjiglesias — APPROVED** [review]

File: source/learning-wazuh/vuln-detection.rst [hunk 5]
```diff
@@ -319,26 +319,26 @@ The results should look like this:
...
+   can be also enabled by using the following Wazuh API endpoint:
+   :ref:`PUT /manager/api/config<api_reference>`.
 
-  .. code-block:: console
-
-      [root@wazuh-manager centos]# sed -i 's/config.experimental_features  = false/config.experimental_features  = true/g' /var/ossec/api/configuration/config.js
 
 
-5. Restart the Wazuh API service:
+5. Restart the Wazuh API using the Wazuh-manager service:
```
[the remark this answers is not in the document]
Comment (alberpilot) [reply source/learning-wazuh/vuln-detection.rst:329]: In addition to this, please consider using the `` format for commands, file names, and, in this case, service names.

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

File: source/learning-wazuh/vuln-detection.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/learning-wazuh/vuln-detection.rst]: done

File: source/installation-guide/compatibility_matrix/index.rst [hunk 6]
```diff
@@ -111,18 +111,13 @@ The compatibility between Wazuh agent and Wazuh manager is guaranteed when the W
...
-The API requires the same ``major.minor`` version as the Wazuh manager in order to be compatible.
-
 
 API and Kibana app
 ^^^^^^^^^^^^^^^^^^
 
 The Wazuh app for Kibana requires compatibility between two different products:
 
-  - With the **Wazuh API**, it requires the same ``major.minor`` version.
+  - With the **Wazuh API**, which comes installed by default. It requires the same ``major.minor`` version.
```
[the remark this answers is not in the document]
Comment (CarlosRS9) [reply source/installation-guide/compatibility_matrix/index.rst]: done

**Review by alberpilot — APPROVED** [review]
LGTM

**Pull request merged**